### PR TITLE
Button Profiles - Missing games on the list

### LINKED ISSUE #187: Button Profiles - Missing games on the list
There are some missing games on the [button profiles list](https://github.com/libretro/mame/blob/master/src/osd/modules/input/input_retro.cpp)

Capcom CPS-1 and CPS-2 6-button fighting games : 

Parent roms -->
- sfz2al
- vhunt2
- redearth
- "starglad" should be removed from this list as it has a [4 buttons layout](http://gamesdbase.com/Media/SYSTEM/Arcade/commands/Commands_Star_Gladiator_-_1996_-_Capcom.htm) 

This list should be renamed "6-button fighting games" with those games added to that list : 

- astrass
- brival
- btlkroad
- dragoona
- fghthist
- fgtlayer
- groovef
- kaiserkn

### PR DESCRIPTION
**Pull request by Immersion95**
Changes suggested here : #187

Added to the 6-button list : 
sfz2al
vhunt2
redearth
astrass
brival
btlkroad
dragoona
fghthist
fgtlayer
groovef
kaiserkn

Removed : starglad (wrongly put in the 6-button list)

### CODE REVIEW

File: src/osd/modules/input/input_retro.cpp [hunk 1]
```diff
@@ -338,9 +342,26 @@ void Input_Binding(running_machine &machine)
...
+              !core_stricmp(machine.system().name, "dragoona") ||
+              !core_stricmp(machine.system().parent, "dragoona") ||
+              !core_stricmp(machine.system().name, "fghthist") ||
+              !core_stricmp(machine.system().parent, "fghthist") ||
+              !core_stricmp(machine.system().name, "fgtlayer") ||
+              !core_stricmp(machine.system().parent, "fgtlayer") ||
+              !core_stricmp(machine.system().name, "groovef") ||
+              !core_stricmp(machine.system().parent, "groovef") ||
+              !core_stricmp(machine.system().name, "kaiserkn") ||
+              !core_stricmp(machine.system().parent, "kaiserkn") ||
```
Comment (tcamargo) [inline src/osd/modules/input/input_retro.cpp]: remove the trailing || from the line in the test condition

**Comment by tcamargo** [comment]
I will merge it after mame 0.223.

**Comment by Immersion95** [comment]
Great.

I only added the missing games from the 6-button list but I would also like to add other 4-button games layout (jojo, powerinstinct, etc...). Is that OK ? It would only be layout that were originally on the official ports.

**Comment by tcamargo** [comment]
You can add those changes to the same PR.

**Pull request closed without merging**